hudi-sync/hudi-hive-sync/pom.xml

```diff
@@ -131,6 +131,14 @@
       <version>${hive.version}</version>
     </dependency>
 
+    <!-- AWS Glue -->
```

Comment:
how about we move this to `hudi-aws` and keep the aws specific integration there?